fvm/src/trace/mod.rs

```diff
@@ -0,0 +1,26 @@
+use fvm_shared::address::Address;
+use fvm_shared::econ::TokenAmount;
+use fvm_shared::encoding::RawBytes;
+use fvm_shared::{ActorID, MethodNum};
+
+use crate::call_manager::InvocationResult;
+use crate::kernel::Result;
+
+/// Execution Trace, only for informational and debugging purposes.
+pub type ExecutionTrace = Vec<ExecutionEvent>;
+
+#[derive(Clone, Debug)]
+pub enum ExecutionEvent {
```

Comment:
Does this not need to derive serialize so that can filecoin-ffi can CBOR-serialize it over the FFI boundary? (Same for SendParams).